contribs/emissions/src/main/java/org/matsim/contrib/emissions/utils/EmissionUtils.java

```diff
@@ -44,6 +48,8 @@
 public class EmissionUtils {
 	private static final Logger logger = Logger.getLogger(EmissionUtils.class);
 
+	@Deprecated // yyyy we don't like static fields, especially not with mutable material.  Pls explain why this one here is
```

Comment:
Hi Kai, i've actually been thinking that we should anyway change the way pollulants are read in. Currently, not all possible pollutants are recorded, only the ones that IIhab was originally interested in. I would propose reading the list of pollutants directly from the hbefa files. We need this anyway for some work we are doing, and it would also solve this issue you highlight here.